modulemd/modulemd-packager-v3.c

```diff
@@ -1628,3 +1628,146 @@ modulemd_packager_v3_parse_module_components (yaml_parser_t *parser,
 
   return TRUE;
 }
+
+
+gboolean
+modulemd_packager_v3_emit_yaml (ModulemdPackagerV3 *self,
+                                yaml_emitter_t *emitter,
+                                GError **error)
+{
+  MODULEMD_INIT_TRACE ();
+  g_autoptr (GError) nested_error = NULL;
+  gsize i;
+  g_autoptr (GPtrArray) keys = NULL;
+  gboolean ret;
+
+  /* Emit document headers */
+  if (!modulemd_yaml_emit_document_headers (
+        emitter, MODULEMD_YAML_DOC_PACKAGER, MD_PACKAGER_VERSION_THREE, error))
+    {
+      return FALSE;
+    }
+
+  /* Start data: */
+  EMIT_MAPPING_START (emitter, error);
+
+  /* TODO: emit data */
```

Comment:
You probably want to drop the TODO here.

Reply:
Yup. That's a leftover.